validate/automated/nodectlUpgrade.md

```diff
@@ -0,0 +1,712 @@
+---
+title: upgrading with nodectl
```

Comment:
This whole page is super long and detailed but the actual process of upgrading is very easy. Can we maybe add a quick start option at the top so that most people can skip all the detail? 

All you need to do is run `sudo nodectl upgrade` and follow the prompts.

Reply:
Added `quick start` for both `upgrade` and `upgrade_nodectl`.